packages/lib-ripple/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@dfns/lib-ripple",
```

Comment:
let's call this `lib-xrpl`, there was an old deprecated package called [ripple-lib](https://www.npmjs.com/package/ripple-lib), let's not cause any confusion what we integrate with.